apix-impl/src/main/java/eu/xenit/apix/alfresco/dictionary/DictionaryService.java

```diff
@@ -109,9 +111,19 @@ public TypeDefinition GetTypeDefinition(eu.xenit.apix.data.QName qname) {
     public PropertyDefinition GetPropertyDefinition(eu.xenit.apix.data.QName qname) {
         return propertyService.GetPropertyDefinition(qname);
     }
+    @Override
+    public Properties getProperties() {
+        return propertyService.getProperties();
+    }
 
     @Override
     public AspectDefinition GetAspectDefinition(eu.xenit.apix.data.QName qname) {
         return aspectService.GetAspectDefinition(qname);
     }
+
+    @Override
+    public Aspects getAspects() {
+    return aspectService.getAspects();
```

Comment:
I find it peculiar that even though we already have an AspectService, the aspect are accessed via DictionaryService, which does nothing but relay to the AspectService... However, I checked and this has been in the code since 2015, so it's not blocking here :)